test/test_automl/test_automl.py

```diff
@@ -43,8 +43,11 @@ def setUp(self):
         self.automl._delete_output_directories = lambda: 0
 
     def test_refit_shuffle_on_fail(self):
-        output = os.path.join(self.test_dir, '..', '.tmp_refit_shuffle_on_fail')
-        context = BackendContext(output, output, False, False)
+        tmp = os.path.join(self.test_dir, '..', '.tmp._refit_shuffle_on_fail')
```

Comment:
Could you please create a helper function for these lines, which returns a backend object?